rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewTreeObserverPreDrawEventOnSubscribe.java

```diff
@@ -0,0 +1,39 @@
+package com.jakewharton.rxbinding.view;
+
+import android.view.View;
+import android.view.ViewTreeObserver;
+import com.jakewharton.rxbinding.internal.MainThreadSubscription;
+import rx.Observable;
+import rx.Subscriber;
+
+import static com.jakewharton.rxbinding.internal.Preconditions.checkUiThread;
+
+final class ViewTreeObserverPreDrawEventOnSubscribe
+    implements Observable.OnSubscribe<ViewTreeObserverPreDrawEvent> {
+  private final View view;
+
+  public ViewTreeObserverPreDrawEventOnSubscribe(View view) {
+    this.view = view;
+  }
+
+  @Override public void call(final Subscriber<? super ViewTreeObserverPreDrawEvent> subscriber) {
+    checkUiThread();
+
+    final ViewTreeObserver.OnPreDrawListener listener = new ViewTreeObserver.OnPreDrawListener() {
+      @Override public boolean onPreDraw() {
+        if (!subscriber.isUnsubscribed()) {
+          subscriber.onNext(ViewTreeObserverPreDrawEvent.create(view));
+        }
+        return false;
```

Comment:
The return value here is very important for behavior. It shouldn't just always return `false`.